PostToExternalServices.py

```diff
@@ -0,0 +1,156 @@
+import json
+import logging
+import os
+
+import boto3
+import requests
+import tweepy
+
+AUTH0_BASE = "https://calligre.auth0.com"
+AUTH0_API = "%s/api/v2/" % AUTH0_BASE
```

Comment:
Uh, what I mean is you're hitting routes such as `auth0.com/api/v2//oauth`. If it's treated as a string anywhere by auth0, they might do an `is_valid` check of some sort that could fail due to the double-slash.